README.md

```diff
@@ -52,7 +52,7 @@ class HelloWorld(Resource):
 ```
 ### 2. Define handler
 ```python
-# simple_resource_handler.py
+# simple_resource.py
```

Comment:
I just saw not working line is also present in the README.md examples:
```python
return LambdaFWException().to_dict()
```

It would be good to update them as well, but it is a really minor thing 😉